src/Protocol/Smtp.php

```diff
@@ -273,10 +273,7 @@ public function data($data)
         $this->_send('DATA');
         $this->_expect(354, 120); // Timeout set for 2 minutes as per RFC 2821 4.5.3.2
 
-        // Ensure newlines are CRLF (\r\n)
-        $data = str_replace("\n", "\r\n", str_replace("\r", '', $data));
-
-        foreach (explode(self::EOL, $data) as $line) {
+        foreach (preg_split('/\R/', $data) as $line) {
```

Comment:
in this case no this is not,
with the 2 str_replace
\r => '' (0 eol)
\r\n => \r\n (1 eol)
\n => \r\n (1 eol)
with my preg_split all are 1 eol,
but if my preg_split is unsafe, i'm pretty sure the str_replace is also

Please prove with an exemple that it's insecure :)